package.json

```diff
@@ -9,6 +9,7 @@
     "hello-engine": "node ./src/hello-engine/hello-engine.js",
     "hello-data": "node ./src/hello-data/hello-data.js",
     "hello-visualization": "webpack-dev-server",
+    "line-chart": "webpack-dev-server",
```

Comment:
Maybe we want to keep the naming as in:
- `hello-scatterplot`, and
- `hello-linechart`?